tests/e2e/hcp_machine_pool_test.go

```diff
@@ -1028,6 +1100,18 @@ var _ = Describe("HCP MachinePool", ci.Day2, ci.FeatureMachinepool, func() {
 				mpService.Destroy()
 			}
 		})
+
+		It("worker disk size field - [id:76345]", ci.Low, func() {
```

Comment:
This shows inactive test case in polarion

Reply:
This case also needs to be checked for rosa sts cluster as well

Reply:
+1

Reply:
Done